src/pages/commits/BuildVariantAccordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Disclaimer } from "@leafygreen-ui/typography";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariants: {
+    displayName: string;
+  }[];
+}
+export const BuildVariantAccordion: React.FC<Props> = ({ buildVariants }) => (
+  <ColumnContainer>
+    {buildVariants.map(({ displayName }) => (
+      <AccordionContainer key={displayName}>{displayName}</AccordionContainer>
+    ))}
+  </ColumnContainer>
+);
+
+const AccordionContainer = styled(Disclaimer)`
+  margin-top: 20px;
+  color: ${gray.dark2};
+  font-size: 14px;
+  width: 124px;
+  display: flex;
+  justify-content: flex-start;
+  word-break: break-word;
```

Comment:
Also are these flex properties necessary. They dont seem to affect the styling much when they are removed.